src/main/java/org/jitsi/jigasi/transcription/LocalJsonTranscriptHandler.java

```diff
@@ -205,10 +205,18 @@
     public final static String JSON_KEY_TYPE = "type";
 
     /**
-     * This field stores the value of the type of the muc message as a string
+     * This field stores the value of the type of the muc message  for
+     * a transcription result to be sent.
      */
     public final static String JSON_VALUE_TYPE = "transcription-result";
```

Comment:
to be consistent, we could rename this variable to something like JSON_VALUE_TYPE_TRANSCRIPTION_RESULT